### ISSUE TITLE
WinRTCompiler fails to build in VS2013

### ISSUE THREAD

**Issue by josh-coronalabs** [stammen/angleproject#5]
How to reproduce:
1) Have only Visual Studio 2013 with WP8 support included. (Includes 2012 toolset.)
2) Open the "winrtcompiler.sln" in Visual Studio 2013.
3) Click OK when it asks you to update it to 2013.
4) Build a "Release" version.

Result:
I get several compiler errors like this...
   error C2245: non-existent member function 'Platform::Details::Heap::Allocate' specified as friend

Notes:
- I'm able to build the Angle project for Windows Phone 8 just fine for ARM and Win32.
- I've noticed that the project configuration's "Additional #using directories" contains %28 and %29 where parenthesis should by in the "Program Files (x86)" part of the path.  Changing them to parenthesis did not solve the problem.
- I verified that files "Platform.winmd" and "Windows.winmd" are in the directories specified under "Addition #using directories".
- I've noticed that the release build configuration has "Consume Windows Runtime Extensions" set to YES for release, but NO for debug.

I don't have Visual Studio 2012 available to me at the moment and will look into attempting a build with it later next week.

**Comment by josh-coronalabs** [comment]
Additional Note:
I used to be able to build the old glslPrecompiler.exe in Visual Studio 2013 just fine with the changes I've mentioned in issue #3 (link below) and adding preprocessor definitions COMPILE_SHADER, USE_FEATURE_LEVEL_9_1, and ANGLE_ENABLE_D3D11.
   https://github.com/stammen/angleproject/issues/3

But that said, I merely tested if that glslPrecompiler was able to convert/compile shader language into byte code.  I did not verify if the compiled shader actually worked.
